README.md

```diff
@@ -2,62 +2,107 @@
 
 This is a CosmWasm smart contract to manage Curve Leverage Lending Bot smart contract on EVM chain written in Vyper.
 
-Users can create a crvUSD loan by deposit their token into a Vyper smart contract with leverage option on EVM chain.
+Users can create a Curve Leverage Lending Bot by depositing their tokens into a Vyper smart contract with options for leverage on the EVM chain.
 
-A scheduler or script fetch events from the Vyper smart contract and run `repay_bot` function with the parameters via Compass-EVM on high risk or expiration.
+A scheduler or script fetches events from the Vyper smart contract and runs `repay_bot` function with the parameters via Compass-EVM on high risk or expiration.
 
-And then, the Vyper smart contract will repay the bot.
+The Vyper smart contract will then repay the bot.
 
 ## ExecuteMsg
 
 ### RepayBot
 
 Run `repay_bot` function on Vyper smart contract.
```

Comment:
The original version is correct. This function has only one parameter in a struct array(vector) form.